starter_code/views/beers.hbs

```diff
@@ -0,0 +1,8 @@
+
+{{!-- When looping through items in each, you can optionally reference the current loop index via {{@index}} --}}
+
+ <ul>
+  {{#each beers}}
+     {{> beerPartial this}}
+  {{/each}}
```

Comment:
Pas besoin de mettre this ici, comme on est dans une boucle, le partial va le trouver tout seul :)
```js
  {{#each beers}}
     {{> beerPartial }}
  {{/each}}
```